### ISSUE TITLE
Zen-coding 0.7 for dreamweaver "Indent Code" conflict

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#231]
```
With new feature expand Abbreviation with "tab" key in Dreamweaver CS5 you 
can't select lines and "Indent" them altogether.

So, natural "Indent Code" command is blocked by zen-coding expand now :(

Solutions:
1) Don't expand selected text at all (Actually, dunno when its useful)
2) If text is selected, check for tabs OR spaces inside of it and don't expand 
in that's case.

Thx
```

---

Original issue reported on code.google.com by `sublimey...@gmail.com` on 22 Dec 2010 at 6:03

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I suggest changing the priority to high, because it's breaks one of the basic 
functionalities in DW. Indenting multiple lines of code is also necessary for 
every coder who wants his code to be readable.

I fixed it temporary by removing the command to use Tab for expanding. I 
attached the file if someone wants to use it until a fix is released.

Please fix this ASAP. Thanks for your great efforts.
```

---

Original comment by `mahertheclaw` on 28 Dec 2010 at 9:00

Attachments:
- [Zen Coding.zxp](https://storage.googleapis.com/google-code-attachments/zen-coding/issue-231/comment-2/Zen Coding.zxp)

**Comment by GoogleCodeExporter** [comment]
```
I think it can be fixed via checking selection for correct abbreviation (so 
“<div>” will not expand into “<<div></<div>” because of “<”, and 
whitespace except text nodes will prevent expanding automatically). Will such 
a fix satisfy you?
```

---

Original comment by `GreLIm...@gmail.com` on 29 Dec 2010 at 1:06
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
The best solution is to not expand abbreviation at all if there's selection and 
provide alternative action shortcut (like Ctrl+E) for expanding selection. I'm 
doing the same thing in Eclipse plugin
```

---

Original comment by `serge....@gmail.com` on 30 Dec 2010 at 3:15

**Comment by GoogleCodeExporter** [comment]
```
Issue 236 has been merged into this issue.
```

---

Original comment by `GreLIm...@gmail.com` on 9 Jan 2011 at 12:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 0.7.1.
```

---

Original comment by `GreLIm...@gmail.com` on 10 Jan 2011 at 12:08
- Changed state: **Verified**
